designer/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/ProcessModelMigrator.scala

```diff
@@ -11,15 +12,23 @@ import pl.touk.nussknacker.ui.process.repository.ScenarioWithDetailsEntity
 
 final case class MigrationResult(process: CanonicalProcess, migrationsApplied: List[ProcessMigration]) {
 
-  def toAutomaticProcessUpdateAction(processId: ProcessId, labels: List[ScenarioLabel]): AutomaticProcessUpdateAction =
-    AutomaticProcessUpdateAction(
-      processId = processId,
-      canonicalProcess = process,
-      labels = labels,
-      increaseVersionWhenJsonNotChanged = true,
-      forwardedUserName = None,
-      migrationsApplies = migrationsApplied
-    )
+  def toAutomaticProcessUpdateAction(
+      processId: ProcessId,
+      labels: List[ScenarioLabel]
+  ): Option[AutomaticProcessUpdateAction] = {
```

Comment:
I'm not sure if we shouldn't allow producing MigrationResult with Nil in `migrationsApplied`. WDYT?